CompilerSource/backendflat/Background.fbs

```diff
@@ -0,0 +1,27 @@
+
+table Image {
+	width:int;
+	height:int;
+	data:[byte];

```

Comment:
Format? If MIME, comment it; if RGBA, use int; if any of these, use a separate message and a `oneof`.

Reply:
I'm confused, are you asking me to add the MIME as a field? I believe we were just sending the raw image data prior, so it's always in the same format. This means the IDE, and not the compiler, requires an image loader. Please elaborate and clarify further.